model_compression_toolkit/core/common/quantization/quantization_params_generation/lut_kmeans_params.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 # ==============================================================================
 
+from typing import Dict
```

Comment:
Just out of curiosity, how do you get to change the typehints on this unrelated file?
(In general, I think that it is preferred to not "fix" things that are unrelated to the PR because it overloads the changes and messes up the file's tracking, the change would appear with a commit message that is completely unrelated to the change that is made in the file)